xrootd/client/file.go

```diff
@@ -120,6 +121,14 @@ func (f *file) Stat(ctx context.Context) (xrdfs.EntryStat, error) {
 	return resp.EntryStat, nil
 }
 
+// VerifyWriteAt writes len(p) bytes from p to the file at offset off using crc32 verification.
+// TODO: note that verifyw is not supported by the XRootD server.
```

Comment:
please add an empty doc line before the `TODO` to separate it from the doc header:

```go
// VerifyWriteAt writes len(p) bytes from p to the file at offset off using crc32 verification.
//
// TODO: note that verifyw is not supported by the XRootD server.
// See https://github.com/xrootd/xrootd/issues/738 for the details.
```

this will render better on godoc.org.